.github/workflows/test-dbt-installation-notify-job-statuses.yml

```diff
@@ -113,3 +114,16 @@ jobs:
 
     secrets:
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
We should rename this to `SLACK_DEV_CORE_ALERTS` to match other workflows and reduce confusion for future us.  It needs to be renamed in this workflow and also in the settings on the repo.

Reply:
It needs to be added to the `actions` repo.  

The secret does exist in the adapters repo right now but since these tests are triggered/run in this repo, it won't have access to the secret in the other repo.

Reply:
Confirming that @nathaniel-may added the secrets to this repo and resolving.